openslides/motions/access_permissions.py

```diff
@@ -34,13 +30,19 @@ def get_restricted_data(self, full_data, user):
             data = full_data
         else:
             data = full_data.copy()
-            for i, field in enumerate(self.get_comments_config_fields()):
-                if not field.get('public'):
-                    try:
+            if isinstance(config['motions_comments'], list):
```

Comment:
You do not have to check the integrity of the config value here any more. You checked it already at input time.